composer.json

```diff
@@ -1,30 +1,44 @@
 {
-  "name": "wpml/page-builders-cornerstone",
-  "description": "A library used by WPML to handle Cornerstone",
-  "type": "library",
-  "license": "GPL-3.0-or-later",
-  "authors": [
-    {
-      "name": "OnTheGoSystems",
-      "email": "hello@wpml.org"
+    "name": "wpml/page-builders-cornerstone",
+    "description": "A library used by WPML to handle Cornerstone",
+    "type": "library",
+    "license": "GPL-3.0-or-later",
+    "authors": [
+        {
+            "name": "OnTheGoSystems",
+            "email": "hello@wpml.org"
+        }
+    ],
+    "autoload": {
+        "classmap": [
+            "src/"
+        ]
+    },
+    "autoload-dev": {
+        "classmap": [
+            "tests/phpunit/util/"
+        ]
+    },
+    "require": {
+        "wpml/page-builders": "dev-wpmlcore-5811"
```

Comment:
@strategio no, I have changes in this page-builders branch. I will change after everything is tested.

Reply:
@adrianosferreira, lastly I reduced the cross-dependency of PB packages because it was almost impossible to run `composer install` without conflicts (see https://onthegosystems.myjetbrains.com/youtrack/issue/wpmlpb-155).

What we need to do is adding this dependency in `require-dev` so it does not cause problems when we install the packages in Core (and we only need it for tests with the base classes). That way, you can decide which branch you want for each PB package in WPML Core.

If you don't have objection, I will remove this dependency and instead I will create a test branch in Core (i.e. `wpmlcore-5811-test`) with the featured packages.

Reply:
@strategio sure, it is ok.